R/class.R

```diff
@@ -238,8 +238,8 @@ is_class <- function(x) inherits(x, "S7_class")
 
 # Object ------------------------------------------------------------------
 
-#' @param .parent,... Parent object and named properties used to construct the
-#'   object.
+#' @param .parent,... Parent S7 object and named properties used to construct
```

Comment:
This isn't necessarily an S7 object, because the class might extend a base class.